pkgs/applications/graphics/gimp/default.nix

```diff
@@ -115,6 +115,7 @@ in stdenv.mkDerivation rec {
     libwebp
     libheif
     python
+    python2.pkgs.pygtk
```

Comment:
Done

Reply:
That does not really say much. Maybe something like “Duplicated here because python.withPackages does not expose the dev output with pkg-config files”

Reply:
Done